src/components/data-pane/wildcard-field-drop-zone.scss

```diff
@@ -0,0 +1,23 @@
+.drop-zone {
```

Comment:
1) Instead of declaring a totally new CSS, can this extend some style of Field

2) The old version of Voyager has nicer CSS here
  - We should rounded corner like normal Field
  - It should be semi-transparent